### DISCUSSION TITLE
Timeframe for 2.0.0

### DISCUSSION THREAD

**Discussion by paulo-ferraz-oliveira** [ninenines/gun#288] [General]
Opening the discussion as per #268 and [@essen's comment](https://github.com/ninenines/gun/issues/268#issuecomment-915242853)... I know you're busy, but are there plans to release 2.0.0 soon? Thanks again.

**Comment by essen** [comment]
Working on it, right now waiting for https://github.com/ninenines/gun/pull/243#discussion_r824856311 and also been merging relevant Cowlib PRs and updating a few modules. On the Cowlib front the cookie tests have changed upstream so I need to figure out how to integrate them again.

**Reply by paulo-ferraz-oliveira** [reply]
Great stuff. Many thanks.
